src/main/java/com/seungwooryu/woostagram/post/domain/Comment.java

```diff
@@ -2,27 +2,38 @@
 
 import com.seungwooryu.woostagram.common.domain.BaseEntity;
 import com.seungwooryu.woostagram.user.domain.User;
-import lombok.Getter;
+import lombok.*;
 
 import javax.persistence.*;
 
 @Entity
 @Table(name = "comments")
 @Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Comment extends BaseEntity {
     @Id
     @Column(name = "id", unique = true, nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "data", nullable = false, length = 1000)
-    private String data;
+    @Column(name = "contents", nullable = false, length = 1000)
+    private String contents;
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name="author_id",referencedColumnName = "id")
-    private User user;
+    private User authorId;
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name="post_id",referencedColumnName = "id")
-    private Post post;
+    private Post postId;
+
+    public Comment(String contents, User authorId, Post postId) {
+        this.contents = contents;
+        this.authorId = authorId;
+        this.postId = postId;
+    }
```

Comment:
정적 팩토리 메서드를 활용하면 좋을듯요.
[참고 링크](https://velog.io/@meme2367/%EC%9D%B4%ED%8E%99%ED%8B%B0%EB%B8%8C-%EC%9E%90%EB%B0%94-%EC%95%84%EC%9D%B4%ED%85%9C-1-%EC%83%9D%EC%84%B1%EC%9E%90-%EB%8C%80%EC%8B%A0-%EC%A0%95%EC%A0%81-%ED%8C%A9%ED%86%A0%EB%A6%AC-%EB%A9%94%EC%84%9C%EB%93%9C%EB%A5%BC-%EC%A0%9C%EA%B3%B5%ED%95%98%EB%9D%BC)
```suggestion
private Comment(String contents, User user, Post post) {
        this.contents = contents;
        this.user = user;
        this.post = post;
    }

public static Comment of(String contents, User user, Post post) {
    return new Comment(contents, user, post);
}
```